src/AccessibilityInsights.SharedUx/Utilities/ExtensionMethods.cs

```diff
@@ -123,7 +126,7 @@ public static void CopyStringToClipboard(this StringBuilder sb)
         /// <param name="isLiveMode">Is node for live mode</param>
         public static HierarchyNodeViewModel GetRootNodeHierarchyViewModel(this ElementDataContext dc, bool showAncestry, bool showUncertain, bool isLiveMode)
         {
-            if (dc.RootElment != null && dc.RootElment.Properties != null && dc.RootElment.Properties.Count != 0 && dc.Element != null)
+            if (dc?.RootElment?.Properties != null && dc.RootElment.Properties.Count != 0 && dc.Element != null)
```

Comment:
>RootElment [](start = 20, length = 10)

I opted to not throw if dc is null, but just treat it as though no element exists. Happy to be more aggressive here if desired